SitePackage.lua

```diff
@@ -278,4 +278,10 @@ hook.register("isVisibleHook", visible_hook)
 
 sandbox_registration{
     get_avail_memory = get_avail_memory,
+    math = {
```

Comment:
these are already in the sandbox? That's what https://github.com/TACC/Lmod/pull/453 does.

You only need to add custom functions here which are not yet added the `sandbox.lua` of Lmod itself.

Reply:
I was not aware of your PR to Lmod. Then this indeed not necessary.